Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM mysql:latest
+MAINTAINER random-user@email.com
+
+ENV MYSQL_USER myuser
```

Comment:
🛑 **gitleaks** has detected a secret with rule-id `` in commit fac4ab999cdd91f8428b160ba3b19749cad4bbf5
If this secret is a true positive, please rotate the secret ASAP and rebase the commit containing the secret.